src/main/io/ledstrip.h

```diff
@@ -143,8 +143,8 @@ typedef enum {
 typedef enum {
     LED_PROFILE_RACE = 0,
     LED_PROFILE_BEACON,
-#ifdef USE_LED_STRIP_STATUS_MODE
-    LED_PROFILE_STATUS,
+#ifdef USE_LED_STRIP_DEFAULT_MODE
+    LED_PROFILE_DEFAULT,
```

Comment:
This is still not right

Reply:
added a commit to change `LED_PROFILE_DEFAULT` and similar to `LED_MODE_DEFAULT`

Reply:
Instead maybe we use should turn around and use `USE_LED_STRIP_RACE_MODE` so we use `LED_MODE_DEFAULT` by default and `LED_MODE_RACE` if `USE_LED_STRIP_RACE_MODE` was defined.

Please note the change from `PROFILE` to `MODE`.

Reply:
Does this solve the problem?
```
typedef enum {
    LED_USER_MODE_RACE = 0,
    LED_USER_MODE_BEACON,
#ifdef USE_LED_STRIP_DEFAULT_MODE
    LED_USER_MODE_DEFAULT,
#endif
    LED_USER_MODE_COUNT
} ledUserMode_e;
```
Note I can't use `LED_MODE_COUNT` here; it won't compile since there are internal LED modes with names like `LED_MODE_HORIZON`  that are counted with `LED_MODE_COUNT`.